TrailBlaze/Networking/Interceptor.m

```diff
@@ -12,25 +12,25 @@
 @implementation Interceptor {
     
 }
-//static float etaDifferenceThreshold = 2;
-static NSMutableArray *etas;
+static float etaDifferenceThreshold = 692;
```

Comment:
Could be good to add a comment using `///` (triple slash) here to document how you arrived at 692. Up to you, though! 

`//` is used for comments while `///` is used for documentation 😄